File: BackEnd/src/main/java/com/example/backend/data/viewModels/AuthModel.java
package com.example.backend.data.viewModels;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.Entity;
import javax.persistence.Table;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AuthModel {

    private String login;
    private String password;


}

File: Infrastructure/ImpInfCommon/Data/Other/AuthModel.cs
using System.ComponentModel.DataAnnotations;

namespace ImpInfCommon.Data.Other
{
    public class AuthModel
    {
        [Required(ErrorMessage = "Поле логина должно быть заполнено")]
        public string Login { get; set; }
        [Required(ErrorMessage = "Поле пароля должно быть заполнено")]
        public string Password { get; set; }
    }
}
